rotkehlchen/db/updates.py

```diff
@@ -372,7 +372,7 @@ def check_for_updates(self, updates: Sequence[UpdateType] = tuple(UpdateType)) -
     def _check_for_last_version(cursor: 'DBCursor', update_type: UpdateType) -> int:
         """This method checks the database for the last local version of `update_type`"""
         found_version = cursor.execute(
-            'SELECT value FROM key_value_cache WHERE name=?',
+            'SELECT value FROM settings WHERE name=?',
```

Comment:
This is really bad that the change was not detected. Why did the tests here not fail before? 

Can you add a test that uses this so that this thing actually works and is covered?